index.html

```diff
@@ -10469,16 +10469,30 @@ <h3>application/ld+json</h3>
             It is RECOMMENDED that profile URIs are dereferenceable and provide
             useful documentation at that URI. For more information and background
             please refer to [[RFC6906]].</p>
-          <p>This specification defines three values for the <code>profile</code> parameter.
-            To request or specify <a href="#expanded-document-form">expanded JSON-LD document form</a>,
-            the URI <code>http://www.w3.org/ns/json-ld#expanded</code> SHOULD be used.
-            To request or specify <a href="#compacted-document-form">compacted JSON-LD document form</a>,
-            the URI <code>http://www.w3.org/ns/json-ld#compacted</code> SHOULD be used.
-            To request or specify <a href="#flattened-document-form">flattened JSON-LD document form</a>,
-            the URI <code>http://www.w3.org/ns/json-ld#flattened</code> SHOULD be used.
-            Please note that, according [[HTTP11]], the value of the <code>profile</code>
-            parameter has to be enclosed in quotes (<code>"</code>) because it contains
-            special characters and, if multiple profiles are combined, whitespace.</p>
+          <p>This specification defines four values for the <code>profile</code> parameter.</p>
+          <dl>
+            <dt><code>http://www.w3.org/ns/json-ld#expanded</code></dt>
+            <dd>To request or specify <a href="#expanded-document-form">expanded JSON-LD document form</a>.</dd>
+            <dt><code>http://www.w3.org/ns/json-ld#compacted</code></dt>
+            <dd>To request or specify <a href="#compacted-document-form">compacted JSON-LD document form</a>.</dd>
+            <dt><code>http://www.w3.org/ns/json-ld#flattened</code></dt>
+            <dd>To request or specify <a href="#flattened-document-form">flattened JSON-LD document form</a>.</dd>
+            <dt><code>http://www.w3.org/ns/json-ld#framed</code></dt>
+            <dd>To request or specify <a href="#framed-document-form">framed JSON-LD document form</a>.</dd>
+          </dl>
+          <p>
+            When used as a <a data-cite="RFC2048#section-2.2.3">media type paramter</a> [[RFC2048]]
+            in an <a data-cite="rfc7231#section-5.3.2">HTTP Accept header</a> [[RFC7231]],
+            the value of the <code>profile</code> parameter MUST be enclosed in quotes (<code>"</code>) if it contains
+            special characters such as whitespace, when multiple profiles are combined.</p>
+          <p>In addition to the <code>profile</code> values described above,
+            the <code>profile</code> parameter MAY include one additional URL, which is interpreted as
```

Comment:
Do we have a spot for showing the usage of this "one additional URL"? It would be helpful to see this in an example.

For instance, how are the two profile URL's expressed together? As two separate `profile=`'s? or are they delimited somehow as a single value of `profile=`?

Reply:
I thought it was clear that multiple URIs are enclosed within double quotes, but, yes, there should be an example. We can add an examples section for this, as I wouldn't want to muddy the registration itself.

Reply:
Added examples in 2646290098acb11c9ae6cb9e58622a1986286185.

Reply:
Thanks for the example!